nextcloud-aio-helm-chart/readme.md

```diff
@@ -1,6 +1,19 @@
-# You can also install the AIO containers on Kubernetes using this Helm Chart
+# Nextcloud AIO Helm-chart
 
-This is currently beta and not ready yet.
+You can run the containers that are build for AIO with Kubernetes using this Helm chart. This comes with a few downsides, that are discussed below.
+
+### Advantages
+- You can run it without a container having access to the docker socket
+- You can run the containers with Kubernetes
+
+### Disadvantages
+- You lose the AIO interface
+- You lose update notifications and automatic updates
+- You lose all AIO backup and restore features
+- You need to know what you are doing
```

Comment:
Maybe even show that one in bold :laughing:

Reply:
good idea, done! :D